src/app/shared/pricing-definitions/pricing-definition/pricing-definition.component.html

```diff
@@ -9,7 +9,7 @@
           </ng-template>
```

Comment:
Missing padding/margin at the bottom:
<img width="976" alt="2022-01-24_17-44-47" src="https://user-images.githubusercontent.com/25266945/150826450-6d83ade1-a793-4f53-ab3c-531021c3ba10.png">